product-service/handlers/getProductsList.js

```diff
@@ -0,0 +1,23 @@
+import productList from './productList.json'
+
+export const getProductsList = async event => {
+  try {
+    return {
+        statusCode: 200,
+        headers: {
+            // "Access-Control-Allow-Credentials" : true // Required for cookies, authorization headers with HTTPS
+            'Content-Type': 'application/json',
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept',
+            'Access-Control-Allow-Methods': 'POST, GET, OPTIONS, PATCH, PUT',
+        },
```

Comment:
It worth moving these headers to the const and use between handlers.

Reply:
changed it